go-controller/pkg/config/utils.go

```diff
@@ -37,7 +37,8 @@ type CIDRNetworkEntry struct {
 // ParseClusterSubnetEntries returns the parsed set of CIDRNetworkEntries passed by the user on the command line
 // These entries define the clusters network space by specifying a set of CIDR and netmasks the SDN can allocate
 // addresses from.
-func ParseClusterSubnetEntries(clusterSubnetCmd string) ([]CIDRNetworkEntry, error) {
+// For layer2 CIDRs, not to check subnet length
+func ParseClusterSubnetEntries(clusterSubnetCmd string, checkHostSubnetLength bool) ([]CIDRNetworkEntry, error) {
```

Comment:
This implementation prevents a flat L2 from having no network CIDR - which I had assumed would be the way for the user to express IPAM will be handled by something other than OVN-K.

When trying this PR without a defined subnet, we get the following error:
```golang
net-attach-def cluster1/ovn is first seen and is invalid: invalid layer2 netconf cluster1: cluster subnet  is invalid: CIDR "" not properly formatted
```

Reply:
same as above